vassal-app/src/main/java/VASSAL/counters/BoundsTracker.java

```diff
@@ -43,6 +43,14 @@ public void clear() {
 
   public void addPiece(GamePiece p) {
     if (p.getMap() != null) {
+      if (p.getParent() != null) {
+        // NB: We track the Stack if there is one. This is because individual pieces
```

Comment:
@uckelman An alternative to writing this comment would be to create a bug in Github and then just reference it there "for more detail". Not sure if you like comments like this in the code or not.